resources/js/Pages/Holidays/Edit.vue

```diff
@@ -66,8 +66,9 @@
           </InertiaLink>
           <button
             type="submit"
-            :disabled="form.processing"
             class="inline-flex justify-center py-2 px-4 text-sm font-medium text-white bg-blumilk-600 hover:bg-blumilk-700 rounded-md border border-transparent focus:outline-none focus:ring-2 focus:ring-blumilk-500 focus:ring-offset-2 shadow-sm"
+            :class="{ 'bg-gray-600 hover:bg-gray-600': form.processing || !form.isDirty }"
```

Comment:
`disabled:opacity-60` will be better than this gray color, because it suggest to user that the button is clickable but it doesn't. It applies to all submit buttons in editing forms.